scripts/deployments.js

```diff
@@ -178,14 +178,13 @@ module.exports = async function(robot) {
 
   robot.hear(/!build (goerli|mainnet) ([0-9a-fA-f]*)/, async msg => {
     const buildInfo = await request({
+      method: 'POST',
       uri: `https://circleci.com/api/v1.1/project/github/JoinColony/colonyDapp/tree/${msg.match[2]}`,
-      qs: {
-        'circle-token': process.env.CIRCLE_CI_API_KEY,
+      auth: {
+        'user': process.env.CIRCLE_CI_API_KEY
```

Comment:
Interesting that the above didn't work. Circle docs said it would be ok 🤔 (https://circleci.com/docs/api/#add-an-api-token). But if it's working I don't really care how in this case :)